subsys/net/ip/6lo.c

```diff
@@ -4,6 +4,7 @@
 
 /*
  * Copyright (c) 2016 Intel Corporation
+ * Copyright (c) 2019 Alexander Wachter
```

Comment:
review the commit title, instead of 'net: ip:'  have 'subys: net:'  or just 'net:' apply that on other commit titles.

Reply:
> have 'subys: net:'

There should never be "subsys: net: " prefix as the "net: " is already the subsystem so there is no need to add duplicate information.
Depending on context, sometimes "net: ip: ..." makes sense but in this case for 6lo stuff, the prefix should probably be just "net: 6lo: ..." or similar (depending on the stuff we are changing in the commit).

Reply:
> There should never be "subsys: net: " prefix as the "net: " is already the subsystem

I have been using that format without any complaints, moreover when other patches touches tests/net, then tests: net: was the prefix, so it differentiates which part in zephyr targeting net domain.